engines/order_management/app/services/order_management/order/updater.rb

```diff
@@ -27,9 +28,6 @@ def update
           update_shipment_state
         end
 
-        update_all_adjustments
-        # update totals a second time in case updated adjustments have an effect on the total
-        update_totals
```

Comment:
The only reason I could see is that one of those totals is needed while updating adjustments but that's not the case AFAIK, right @Matt-Yorkley ?

Reply:
Nope :+1:

Reply:
None of the adjustment updates will use those totals.